crates/devolutions-pedm/src/api/mod.rs

```diff
@@ -53,6 +47,14 @@ mod revoke;
 pub(crate) mod state;
 mod status;
 
+use about::about;
+use elevate_session::elevate_session;
+use elevate_temporary::elevate_temporary;
+use launch::post_launch;
+use revoke::post_revoke;
+use state::{AppState, AppStateError};
+use status::get_status;
```

Comment:
style: I recommend to use `self::` to make it clear we are importing from the current module, but it’s not a big deal.